common/djangoapps/common_initialization/apps.py

```diff
@@ -0,0 +1,66 @@
+"""
+Common initialization app for the LMS and CMS
+"""
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import Error, Tags, register
+
+
+class CommonInitializationConfig(AppConfig):
+    name = 'common_initialization'
+    verbose_name = 'Common Initialization'
+
+
+@register(Tags.compatibility)
```

Comment:
I thought the Django docs [recommend](https://docs.djangoproject.com/en/1.8/topics/checks/#registering-and-labeling-checks) registering checks in code imported from inside the `ready()` method, so the code doesn't get run too soon

Reply:
Oh, reading that makes me agree.  BTW, i think we recently committed a different file using this incorrect pattern: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/theming/apps.py

Reply:
Indeed! @pwnage101 Could you please add a commit to this PR that indents that code under `ready()`? We shouldn't leave anti-patterns in the codebase if it can be helped.

Reply:
@doctoryes let me know what you think about the pattern I just committed in this PR, then i'll add a commit for theming/apps.py

Reply:
@pwnage101 Don't know what the current test failures are - but pattern looks fine to me.